test/fixtures/poro.rb

```diff
@@ -136,10 +136,11 @@ def custom_options
 end
 
 RoleSerializer = Class.new(ActiveModel::Serializer) do
-  cache only: [:name], skip_digest: true
-  attributes :id, :name, :description, :slug
+  cache only: [:name, :slug], skip_digest: true
+  attributes :id, :name, :description
```

Comment:
having the cache options refer to a key was untested. if the only key here were `friendly_id`, it might get weird..